eslzArm/managementGroupTemplates/policyAssignments/DINE-SQLEncryptionPolicyAssignment.json

```diff
@@ -23,15 +23,15 @@
     },
     "variables": {
         "policyDefinitions": {
-            "deploySqlEncryption": "/providers/Microsoft.Authorization/policyDefinitions/36d49e87-48c4-4f2e-beed-ba4ed02b71f5"
+            "deploySqlEncryption": "/providers/Microsoft.Authorization/policyDefinitions/86a912f6-9a06-4e26-b447-11b16ba8659f"
```

Comment:
Have you tested this doesnt cause a breaking change? e.g. deploy from main, then try and deploy this branch?